hedera-node/hedera-app/src/main/java/com/hedera/node/app/info/GenesisNetworkInfo.java

```diff
@@ -38,19 +40,17 @@
  */
 public class GenesisNetworkInfo implements NetworkInfo {
     private final Bytes ledgerId;
-    private final Roster genesisRoster;
     private final Map<Long, NodeInfo> nodeInfos;
 
     /**
      * Constructs a new {@link GenesisNetworkInfo} instance.
      *
-     * @param genesisRoster The genesis roster
+     * @param genesisAddressBook The genesis address book
      * @param ledgerId      The ledger ID
      */
-    public GenesisNetworkInfo(@NonNull final Roster genesisRoster, @NonNull final Bytes ledgerId) {
+    public GenesisNetworkInfo(@NonNull final AddressBook genesisAddressBook, @NonNull final Bytes ledgerId) {
```

Comment:
I don't understand why it shouldn't be Roster here

Reply:
Right now there might be so little extra information in the Hedera Address Book that we can default value the content that is missing and extend a roster to become a full address book.  But if the definition of the Hedera Address Book grows, that is more and more content that has to be defaulted in.   Right now our genesis process is only used in test cases and fill-in-the-gaps fill in the gaps with defaults might work.   But very soon with SPNs we need a real genesis behavior and we can't assume a hard-coded default value for things like the account bound to a node.  The future `genesis_network.json` format will have the ability to set a dynamic account value.  The Roster doesn't have that value and can't be an accurate in-memory representation of the genesis network information. 

Roster --> `AddressBook` is only useful to the platform while we are refactoring and incrementally replacing the usage of the AddressBook in the platform.  After the refactor is complete, the only valid translation is to go from `AddressBook`  -->  Roster.    `AddressBook` here in the code should be replaced with whatever format you will use to translate `genesis_network.json` into an in-memory data structure and from there rosters can be created to be given to the platform.